ui/View.elm

```diff
@@ -10,85 +10,133 @@ import Types exposing (..)
-        , ul [ class "list-group" ] <|
-            [ (radioButton filterName checkedFilter onClickHandler "all") ]
-                ++ List.map (radioButton filterName checkedFilter onClickHandler) filters
-        ]
+filterInfoView : Model -> Html Msg
+filterInfoView model =
+    let
+        filterInfos =
+            [ ( "tree", model.treeFilter )
+            , ( "product", model.productFilter )
+            , ( "version", model.versionFilter )
+            , ( "platform", model.platformFilter )
+            , ( "channel", model.channelFilter )
+            , ( "locale", model.localeFilter )
+            , ( "buildId", model.buildIdFilter )
+            ]
+                |> List.filter (\( _, value ) -> value /= "all" && value /= "")
+                |> List.map
+                    (\( filter, value ) ->
+                        span [ class "badge" ] [ text <| filter ++ ":" ++ value ]
+                    )
+                |> List.intersperse (text " ")
 
+        nbBuilds =
+            List.length model.filteredBuilds
+    in
+        p [ class "well" ] <|
+            (List.concat
+                [ [ text <|
+                        (toString nbBuilds)
+                            ++ " build"
+                            ++ (if nbBuilds == 1 then
+                                    ""
+                                else
+                                    "s"
+                               )
+                            ++ " found. "
+                  ]
+                , (if List.length filterInfos > 0 then
+                    [ text "Filters: " ]
+                   else
+                    [ text "" ]
+                  )
+                , filterInfos
+                ]
+            )
 
-radioButton : String -> String -> (String -> Msg) -> String -> Html Msg
-radioButton filterName checkedFilter onClickHandler filterValue =
-    li [ class "list-group-item" ]
-        [ div
-            [ class "radio"
-            , style [ ( "margin", "0px" ) ]
+
+buildIdSearchForm : Model -> Html Msg
+buildIdSearchForm model =
+    div [ class "form-group" ]
+        [ label [] [ text "Build id" ]
+        , input
+            [ type_ "text"
+            , class "form-control"
+            , placeholder "Eg. 201705011233"
+            , value model.buildIdFilter
+            , onInput <| UpdateFilter << NewBuildIdSearch
             ]
-            [ label []
-                [ input
-                    [ name filterName
-                    , type_ "radio"
-                    , value filterValue
-                    , checked <| checkedFilter == filterValue
-                    , onClick <| onClickHandler filterValue
-                    ]
-                    []
-                , text filterValue
+            []
+        ]
+
+
+filterSelector : List String -> String -> String -> (String -> Msg) -> Html Msg
+filterSelector filters filterName checkedFilter updateHandler =
```

Comment:
Now that the filters are options in a select and not radiobuttons anymore, we might want to rename `checkedFilter` to `selectedFilter`?